README.md

```diff
@@ -13,9 +13,9 @@ If `diagonal` is `true` the first element of the list is `1, 1` otherwise is `1,
 If `diagonal` is `false`, the diagonal values are stored in a vector on the `diag` field.
 Labels can be stored on the field `labels` as an `IndexedArray`.  
 
-In pairwise calculations like `cor()`, saving the result as a `PairwiseListMatrix` is `N(N-1)/2` in space, instead of `N*N`. This is very useful when you need to compare a large number of elements.
```

Comment:
`cor` doesn't return a `PairwiseListMatrix`, you need to write your own version saving the result as a `PairwiseListMatrix`.